eda/setup.py

```diff
@@ -29,6 +29,8 @@
     'phik>=0.12.2,<0.13',
     'seaborn>=0.12.0,<0.13',
     'ipywidgets>=7.7.1,<9.0',  # min versions guidance: 7.7.1 collab/kaggle
+    'shap>=0.41,<0.42',
+    'fastshap>=0.3,<0.4',
```

Comment:
Is fastshap good enough to have as a required dependency? What is the relative advantages of fastshap over shap?

Reply:
For the purposes of explaining a few rows here, `fastshap` is slower than `shap`. I sent an update to remove fastshap backend completely (performance + support concern).